aria/utils/yaml.py

```diff
@@ -48,3 +48,8 @@
 
 finally:
     from ruamel import yaml                                                     # pylint: disable=unused-import
+
+
+# Enables support for writing python unicode class with yaml. This is a Temporary fix which
+# should be replaced by a proper usage of ruamel.
```

Comment:
Please fix this comment to specifically mention [ARIA-429](https://issues.apache.org/jira/browse/ARIA-429), the follow up. Also please use a capital letter for "Python", "Unicode", and all uppercase for "YAML". Also not need to capitalize "Temporary".

Yes, this is very petty. :)